### PR TITLE
test: add node v7

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by popomore** [comment]
+1

**Comment by fengmk2** [comment]
可以发1.0.0了

**Comment by fengmk2** [comment]
我发个 1.0.0

File: test/static.test.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ describe('test/static.test.js', () => {
     before(() => {
       app = mm.app({
         baseDir: 'static-server',
+        customEgg: path.join(__dirname, '../node_modules/egg'),
```
Comment (atian25) [inline test/static.test.js]: 这个需要加上?

**Pull request merged**